packages/template-ui/src/components/ChannelHeader.vue

```diff
@@ -60,6 +91,12 @@ export default {
   padding-top: $navbar-height;
   height: $header-height;
   padding-bottom: $navbar-height;
+  &.full-height {
+    height: auto;
```

Comment:
Per Joana's spec (see https://phabricator.endlessm.com/T32103#899834), I think we may want to _remove_ the background image when expanded, so it's just the solid colour. I'll ask her on Slack just to be sure.

Reply:
Yeah, anyways that's unlikely to happen. The easiest is to remove the header background image when expanded. If we also want to remove the navigation bar image we'll have to emit signals or add state, just for this corner case. I talked with Joana and she is fine removing the header background image only, which shows the solid color. I'm also not spending time on transitions or anything on purpose. Ideally it would be:
- Show more: transition to primary color (bg fade out), then expand.
- Show less: unexpand, then transition back (bg fade in).

This made me realize that using the `clamped` boolean was wrong. What we need is to check if the description is expanded or not in order to change the CSS. There is an `expanded` property in the slot. There is no event for `expanded` like `@clampchange=` in vue-clamp but fortunately the `expanded` property has the [sync modifier](https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier).